experiments/issue453/parser.py

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env python
+
+import logging
+import re
+
+from lab.parser import Parser
+
+def main():
+    parser = Parser()
+    parser.add_pattern("translator_derived_variables", r"Translator derived variables: (\d+)", type=int)
+    parser.add_pattern("translator_axioms", r"Translator axioms: (\d+)", type=int)
```

Comment:
The standard translator parser should parse all output of the form "Translator xxx: yyy". If these two attributes are not picked up automatically, that's a bug in Lab we should fix. Could you check this please? If the attributes are parsed by the standard parser, we could remove this custom parser. Or we keep it for people to see how to add their own parser, but then we should note that the two patterns are redundant.

Reply:
I tested this locally with the change you suggested, it picked up all three attributes correctly (derived variables, aixoms, axioms removed by simplifying). As discussed this morning I'd rather remove the parser since there are already good examples for custom parser in readthedocs.